components/camel-firebase/pom.xml

```diff
@@ -0,0 +1,79 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file needs a ASF license header. You can find from existing pom.xml and copy